Source/Infrastructure/Services/GitHub/Webhooks/Handling/TenantMapper.cs

```diff
@@ -1,14 +1,65 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Dolittle.IO;
+using Dolittle.Lifecycle;
+using Dolittle.Serialization.Json;
 using Dolittle.Tenancy;
 using Octokit;
 
 namespace Infrastructure.Services.Github.Webhooks.Handling
 {
+    [Singleton]
     public class TenantMapper : ITenantMapper
     {
+        const string FileName = "installationToTenantMap.json";
+        const string FilePathEnvName = "GITHUB_INSTALLATION_TENANT_MAP_PATH";
+
+        readonly ISerializer _serializer;
+        readonly IFileSystem _fileSystem;
+        readonly string _filePath;
+
+        Dictionary<long, Guid> _mapping;
+
+        public TenantMapper(ISerializer serializer, IFileSystem fileSystem)
+        {
+            _serializer = serializer;
+            _fileSystem = fileSystem;
+
+            var rootPath = Environment.GetEnvironmentVariable(FilePathEnvName) ?? throw new Exception("GITHUB_INSTALLATION_TENANT_MAP_PATH not set");
```

Comment:
This is hard to write a test for - you're hiding the dependency of Environment. 
Couldn't we just for simplicity say that we have a fixed relative folder `Data`, `Configuration` or similar where we keep things and just rely on mounting that to the correct path. We're doing this for Edge Studio and other things in Continuous Improvement.

Reply:
Which leads me to; we should have specs for this

Reply:
Good points! Having a fixed relative folder - but maybe `.config` like many others use - would be better. There's no need to make it configurable like this when we can just mount something to override it at runtime from wherever.